captain/models/topology.py

```diff
@@ -57,14 +61,15 @@ async def run(self):
     def run_jobs(self, jobs: list[str]):
         for job_id in jobs:
             asyncio.create_task(self.run_job(job_id))
+            self.queued_jobs.add(job_id)
             time.sleep(self.node_delay)
 
     def collect_ready_jobs(self):
         next_jobs: list[str] = []
-        for job_id in cast(list[str], self.original_graph.nodes):
+        for job_id in cast(list[str], self.working_graph.nodes):
             if (
                 job_id not in self.finished_jobs
-                and self.original_graph.in_degree(job_id) == 0
+                and self.working_graph.in_degree(job_id) == 0
```

Comment:
I set this to original_graph to prevent future devs from using this on the working_graph to collect nodes with in-degree 0 during flowchart run.

Reply:
ok, it works as this only executes once. keeping the old change.